bosh/releases/pre_render_scripts/api/cloud_controller_ng/jobs/patch_cc_uploader_ctl.sh

```diff
@@ -1,27 +0,0 @@
-#!/usr/bin/env bash
-
-set -o errexit -o nounset
-
-target="/var/vcap/all-releases/jobs-src/capi/cc_uploader/templates/cc_uploader_ctl.erb"
```

Comment:
As far as I can tell, we can never trigger this file to be used anyway (that is: it's not used with bpm).

Reply:
Squashed history and amended commit message based on these comments.
Commit 5f7cd28c48738106936b93d1ae215ead9c8d412d